homeassistant/components/logi_circle/__init__.py

```diff
@@ -72,23 +72,17 @@
     extra=vol.ALLOW_EXTRA,
 )
 
-LOGI_CIRCLE_SERVICE_SET_CONFIG = CAMERA_SERVICE_SCHEMA.extend(
-    {
-        vol.Required(ATTR_MODE): vol.In([LED_MODE_KEY, RECORDING_MODE_KEY]),
-        vol.Required(ATTR_VALUE): cv.boolean,
-    }
-)
+LOGI_CIRCLE_SERVICE_SET_CONFIG = {
```

Comment:
These schemas aren't used in the entity service helper so we need to pass a schema and not just a dict with voluptuous validators. The entity_id item needs to stay too.

Reply:
🤦 Sorry about that, missed that one. Hero!

Reply:
#49166